hubspot/connection/exc.py

```diff
@@ -17,7 +17,10 @@
 
 class HubspotException(Exception):
     """The base HubSpot error."""
-    pass
+    def __init__(self, message):
+        """ Explicitly makes the message available as the 'message' attribute. """
+        self.message = message
```

Comment:
I tried a few ways of making this ("removing the overriden str() and also removing my explicit override") work but couldn't figure it out.  Maybe with your greater familiarity of the project you can find a better solution than the explicit message override I used.